docs/migrate-to-antrea.md

```diff
@@ -0,0 +1,147 @@
+# Migrate from other CNI to Antrea
+
+This document describes how to migrate from other CNIs to Antrea. Currently, we support migration
+from Calico and Flannel.
+As for Calico, NetworkPolicy conversion is supported conditionally. As for Flannel, only Pod
+networking migration is supported.
+
+NOTE: The migration process is only verified on Kubernetes v1.27.3, Calico v3.26 and Flannel
+v0.22.3. Other versions may not be supported. And the migration process only supports
+Linux Node.
+
+With the help of migration [script](../hack/migrate-to-antrea.sh) and the `antctl migrate` command,
+the migration process is fully automated. You can use one of the following commands to start the migration:
+
+```bash
+# Migrate from Calico to Antrea
+./hack/migrate-to-antrea.sh --from calico
+# Migrate from Flannel to Antrea
+./hack/migrate-to-antrea.sh --from flannel
+```
+
+More details about the migration process are described in the following sections. And you can learn how to use
+the script by running `./hack/migrate-to-antrea.sh --help`.
+
+The migration process is divided into four steps:
+
+1. Install Antrea in the cluster.
+2. If necessary, convert NetworkPolices from other CNI to Antrea.
+3. Restart all Pods in the cluster in-place.
+4. Uninstall the old CNI.
+
+## Install Antrea
+
+The script will check the requirement of CNI migration at first. If we are migrating from Calico,
+the script will list all Calico NetworkPolicy to check if there is any unsupported feature. If
+we are migrating from Flannel, the script will check if Antrea is installed in the cluster.
```

Comment:
don't get why this check is only executed with Flannel. And feel it may be too verbose to list the obvious prechecks here.

Reply:
Removed this sentence.